pkg/scheduler/util/utils_test.go

```diff
@@ -207,3 +211,64 @@ func TestGetContainerPorts(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPodFullName(t *testing.T) {
+	pod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "test",
+			Name:      "pod",
+		},
+	}
+	got := GetPodFullName(pod)
+	expected := fmt.Sprintf("%s_%s", pod.Name, pod.Namespace)
+	if got != expected {
+		t.Errorf("Got wrong full name, got: %s, expected: %s", got, expected)
+	}
+}
+
+func newPriorityPodWithStartTime(priority int32, startTime time.Time) *v1.Pod {
+	return &v1.Pod{
+		Spec: v1.PodSpec{
+			Priority: &priority,
+		},
+		Status: v1.PodStatus{
+			StartTime: &metav1.Time{Time: startTime},
+		},
+	}
+}
+
+func TestGetEarliestPodStartTime(t *testing.T) {
+	currentTime := time.Now()
+	pod1 := newPriorityPodWithStartTime(1, currentTime.Add(time.Second))
+	pod2 := newPriorityPodWithStartTime(2, currentTime.Add(time.Second))
+	pod3 := newPriorityPodWithStartTime(2, currentTime)
+	victims := &api.Victims{
+		Pods: []*v1.Pod{
+			pod1,
+			pod2,
+			pod3,
+		},
+	}
+	startTime := GetEarliestPodStartTime(victims)
+	if !startTime.Equal(pod3.Status.StartTime) {
+		t.Errorf("Got wrong earliest pod start time")
+	}
+}
+
+func TestMoreImportantPod(t *testing.T) {
+	currentTime := time.Now()
+	pod1 := newPriorityPodWithStartTime(1, currentTime)
+	pod2 := newPriorityPodWithStartTime(2, currentTime.Add(time.Second))
+
+	err := fmt.Errorf("Got wrong priority comparison result")
+	pod1Higher := MoreImportantPod(pod1, pod2)
+	if pod1Higher {
+		t.Errorf(err.Error())
+	}
+
+	pod1.Spec.Priority = pod2.Spec.Priority
+	pod1Higher = MoreImportantPod(pod1, pod2)
+	if !pod1Higher {
+		t.Errorf(err.Error())
+	}
```

Comment:
Table driven style may be clearer.

Reply:
There're only two different cases, so I don't use a table to contain them. :)

Reply:
> Table driven style may be clearer.

+1

> There're only two different cases, so I don't use a table to contain them. :)

The two cases have the same pattern which could be refactored into a table-driven style.